pickfirst.go

```diff
@@ -49,31 +49,50 @@ type pickfirstBalancer struct {
 	sc balancer.SubConn
 }
 
+var _ balancer.V2Balancer = &pickfirstBalancer{} // Assert we implement v2
+
 func (b *pickfirstBalancer) HandleResolvedAddrs(addrs []resolver.Address, err error) {
```

Comment:
Why not just `panic("use v2")`?

Reply:
Because it is still called by things that `balancer.Get("pickfirst")` it and wrap.  We need to make sure nobody is calling the V1 API if the V2 API is implemented.  Or remove the V1 API.  Same for the other function.

Reply:
Done